Controller/Adminhtml/Import/Ajax.php

```diff
@@ -75,12 +77,13 @@ public function execute()
             $jsonFactory       = $this->resultJsonFactory->create();
             $storeId           = (int)$this->request->getParam('storeId');
             $chunkId           = (int)$this->request->getParam('chunkId');
+            $orderChunks       = (int)$this->request->getParam('ordersChunks');
```

Comment:
What is this and why is this special for the orders?

Reply:
This is the total amount of order chunks and is used to determine the last step of the orders import at witch point import_end activity should be created. Now looking in to it i think there is no need to assign it to a variable since this request param will be used only one time in a conditional and can be called there directly.